UX-Guide-Metadata/principles/index.html

```diff
@@ -136,7 +136,7 @@ <h2>Discovering Accessible Content</h2>
 			<p>While every user has different accessibility needs, meeting the Web Content Accessibility Guidelines
 				(WCAG) 2.0 requirements, even at level A, is a baseline that ensures a book will be widely accessible.
 				Book providers may therefore wish to create specific search capabilities to permit users to find books
-				that have declared conformance to WCAG at any level. A search filter called Accessible that retrieves all books with metadata that indicates conformance to WCAG Level A, AA and/or AAA.</p>
+				that have declared conformance to WCAG at any level. A search filter called Accessible that retrieves all books with metadata that indicates conformance to WCAG Level A, AA or AAA.</p>
```

Comment:
Shouldn't Accessible be in quotes and noted as an example of how this should work?

> A search filter called "Accessible", for example, that retrieves...

Reply:
Done